lib/matplotlib/axes/_subplots.py

```diff
@@ -121,26 +121,33 @@ def get_gridspec(self):
 
     def update_params(self):
         """update the subplot position from fig.subplotpars"""
-
-        self.figbox, self.rowNum, self.colNum, self.numRows, self.numCols = \
+        self.figbox, _, _, self.numRows, self.numCols = \
```

Comment:
Why keep `self.numRows\Cols` around?

Reply:
Because I don't have a particularly good reason to deprecate them, whereas I'd say rowNum and colNum are actively encouraging people (... the few who'd touch that part of the codebase) to ignore the case of subplots spanning multiple cells.